README.md

```diff
@@ -4,21 +4,33 @@ A simple tool to serve your own discord bot so you can manage an Aternos server
 
 ## Getting Started
 
-1. Git clone this repository 
-1. Install it using either:
+1. Git clone this repository
+2. Install it using either:
    * Copy, paste and execute this command inside the project folder: ```pip install -r requirements.txt```
-   * Alternatively you can create a virtual environement using ```python -m venv venv ``` and then ```source venv/bin/activate``` to finally ```pip install -r  requirements.txt```
-1. Execute using this command inside the project folder : python3 Bot.py
+   * Alternatively you can create a virtual environnement using ```python -m venv venv``` and then ```source venv/bin/activate``` to finally ```pip install -r  requirements.txt```
+3. Execute using this command inside the project folder : ```python3 Bot.py```
+
+**Note:** When running ```Bot.py``` for the first time, you'll be prompted to enter your Aternos account and bot token. You'll need to run ```python3 Bot.py``` again after you finish setting your information.
```

Comment:
Just wanna say thanks, maintaining documentation sucks, and I should've updated it a while ago.